config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
I see what you are saying. It would be more organized and easier to understand.
Thanks Wes! I'll update the PR.